op-e2e/e2eutils/disputegame/helper.go

```diff
@@ -0,0 +1,143 @@
+package disputegame
+
+import (
+	"context"
+	"math"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
+	"github.com/ethereum-optimism/optimism/op-chain-ops/deployer"
+	"github.com/ethereum-optimism/optimism/op-challenger/config"
+	"github.com/ethereum-optimism/optimism/op-challenger/fault"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/challenger"
+	"github.com/ethereum-optimism/optimism/op-service/client/utils"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/holiman/uint256"
+	"github.com/stretchr/testify/require"
+)
+
+const faultGameType uint8 = 0
+const alphabetGameDepth = 4
+
+type Status uint8
+
+const (
+	StatusInProgress Status = iota
+	StatusChallengerWins
+	StatusDefenderWins
+)
+
+var alphaExtraData = common.Hex2Bytes("1000000000000000000000000000000000000000000000000000000000000000")
+var alphabetVMAbsolutePrestate = uint256.NewInt(140).Bytes32()
```

Comment:
This was my error. The character "'" should be decimal 96. The value "140" is in octal.

Reply:
https://github.com/ethereum-optimism/optimism/pull/6369